foreign_cc/private/framework/toolchains/linux_commands.bzl

```diff
@@ -71,10 +69,14 @@ if [ -d "$1" ]; then
   local files=$(find -P $1 \\( -type f -and \\( -name "*.pc" -or -name "*.la" -or -name "*-config" -or -name "*.mk" -or -name "*.cmake" \\) \\))
   IFS=$SAVEIFS
   for file in ${files[@]}; do
-    sed -i 's@'"$2"'@'"$3"'@g' "${file}"
+    local backup=$(mktemp)
```

Comment:
@UebelAndre It appears that these changes have caused apr_util to fail as the "rules.mk" file is not in the sandbox anymore

Reply:
Try with the new changes? I was able to build with them.
```
//python:python2_test                                                    PASSED in 0.5s
//python:python3_test                                                    PASSED in 0.7s
```

Reply:
After rebasing off your branch again, everything passes except for Python on MacOS standalone. Sadly I do not own a Mac so can't really debug it :(

Reply:
Concerning... Seems like a new build succeeded? Is there some flakiness going on here?

Reply:
I have run the following 5 times from the `examples/third_party` directory on my linux machine:

```bash
bazel clean --expunge
bazel test //python:python_tests
```

The tests passed each time. This would indicate the the issue is specifically to do with the standalone MacOS build